temporal-sdk/src/main/java/io/temporal/internal/replay/ReplayWorkflowTaskHandler.java

```diff
@@ -235,6 +236,13 @@ private Result createCompletedWFTRequest(
       }
       completedRequest.setStickyAttributes(attributes);
     }
+    if (result.getSdkFlags().size() > 0) {
```

Comment:
```suggestion
    if (!result.getSdkFlags().isEmpty()) {
```

(I am assuming this is a collection type